### ISSUE TITLE
debian packages :D

### ISSUE THREAD

**Issue by Blackrucat** [WayfireWM/wayfire#1226]
_Please describe what you want Wayfire to do._

**Debian Packages for Wayfire**

_If applicable, give an example: User does X, Y, Wayfire does Z_
**It's better for end users like me, who was a bad PC and don't want to go through every compiling error or whatever it needs to compile. 
I just want WayFire!**

**Comment by soreau** [comment]
You can look at https://github.com/DurandA/wayfire/wiki/Ubuntu-20.04-installation for general dependency list and [wf-install](https://github.com/WayfireWM/wf-install) is a script to build everything wlroots and wayfire.

**Comment by ammen99** [comment]
We do not provide any official packages. I know this is unfortunate, but there are hundreds of distros, we can't make a package for all of them.

**Comment by ammen99** [comment]
> You can look at https://github.com/DurandA/wayfire/wiki/Ubuntu-20.04-installation for general dependency list and [wf-install](https://github.com/WayfireWM/wf-install) is a script to build everything wlroots and wayfire.

Probably already outdated, wlroots needs libseat nowadays ...

**Comment by soreau** [comment]
I created [these packages](http://northfield.ws/wayfire-deb-packages.tar.gz) on 20.04. To install, use `dpkg -i *.deb` followed by `apt-get -f install`.

**Comment by ericonr** [comment]
Working on a physical machine running 20.04 after updating the system (it was ~6 months out of date at least).

xterm was working inside the session too

**Comment by Blackrucat** [comment]
The PPA is not working anymore.

**sudo add-apt-repository ppa:soreau/wayfirewm
sudo apt-get update**

```
Err:16 http://ppa.launchpad.net/soreau/wayfirewm/ubuntu bionic Release         
  404  Not Found [IP: 2001:67c:1560:8008::19 80]
Lendo listas de pacotes... Pronto                            
E: The repository 'http://ppa.launchpad.net/soreau/wayfirewm/ubuntu bionic Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.

```

**Comment by soreau** [comment]
> The PPA is not working anymore.
> 
> **sudo add-apt-repository ppa:soreau/wayfirewm sudo apt-get update**
> 
> ```
> Err:16 http://ppa.launchpad.net/soreau/wayfirewm/ubuntu bionic Release         
>   404  Not Found [IP: 2001:67c:1560:8008::19 80]
> Lendo listas de pacotes... Pronto                            
> E: The repository 'http://ppa.launchpad.net/soreau/wayfirewm/ubuntu bionic Release' does not have a Release file.
> N: Updating from such a repository can't be done securely, and is therefore disabled by default.
> N: See apt-secure(8) manpage for repository creation and user configuration details.
> ```

It is only available for >=20.04, there are no packages for bionic.
